<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="price_box">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="tax_rate">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QPushButton" name="calc_tax_button">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>320</y>
      <width>150</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Tax</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="results_window">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>400</y>
      <width>140</width>
      <height>140</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="light_button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>150</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Light hard on</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_pumping_button">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>240</y>
      <width>150</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Pumping</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="pump_run_time">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>320</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="water_run_time_label">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>320</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>run time</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="pump_delay_time">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>390</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pump_delay_time_label">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>390</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>delay time</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>38</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
